ci/terraform/oidc/mfa-reset-authorize.tf

```diff
@@ -0,0 +1,65 @@
+module "mfa_reset_authorize_role" {
+  source      = "../modules/lambda-role"
+  environment = var.environment
+  role_name   = "mfa-reset-authorize-role"
+  vpc_arn     = local.authentication_vpc_arn
+  #Redis access policy?
+  policies_to_attach = [
+    aws_iam_policy.mfa_reset_storage_token_kms_signing_policy.arn,
+    aws_iam_policy.mfa_reset_authorize_token_kms_signing_policy.arn,
+    module.oidc_txma_audit.access_policy_arn,
+  ]
+}
+
+
+module "mfa_reset_authorize" {
+  source = "../modules/endpoint-module"
+
+  endpoint_name   = "mfa-reset-authorize"
+  path_part       = "mfa-reset-authorize"
+  endpoint_method = ["GET"]
+  environment     = var.environment
+
+  handler_environment_variables = {
+    IPV_AUDIENCE                          = var.ipv_audience
+    STORAGE_TOKEN_SIGNING_KEY_ALIAS       = aws_kms_alias.storage_token_signing_key_alias.arn,
+    IPV_MFA_RESET_TOKEN_SIGNING_KEY_ALIAS = aws_kms_alias.mfa_reset_signing_key_alias.arn,
```

Comment:
Can this variable name match the one in _mfa-reset-storage-token-jwk.tf_:
`MFA_RESET_STORAGE_TOKEN_SIGNING_KEY_ALIAS = aws_kms_alias.mfa_reset_signing_key_alias.name`